localkube.go

```diff
@@ -15,20 +16,28 @@ func (lk *LocalKube) Add(server Server) {
 	lk.Servers = append(lk.Servers, server)
 }
 
-func (lk *LocalKube) Run(args []string, out io.Writer) error {
-	if len(args) < 2 {
+var containerized = flag.Bool("containerized", true, "Whether localkube is inside a container or not")
```

Comment:
Can we auto-detect this by looking at some env vars or inspecting `/proc/self/ns`?